### PR TITLE
* #82: Migrate common tools for implementing a new dialect from the `…

### LINKED ISSUE #82: Migrate common tools for implementing a new dialect from the `virtual-schemas` repository
We should migrate the following parts from the `virtual-schemas` repository to this repository:

https://github.com/exasol/virtual-schemas/tree/master/doc
https://github.com/exasol/virtual-schemas/tree/master/tools
https://github.com/exasol/virtual-schemas/tree/master/model
https://github.com/exasol/virtual-schemas/tree/master/launch

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**
…virtual-schemas` repository.
Closes #82

### CODE REVIEW

File: doc/changes/changes_9.0.0.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Code name:
 * #77: Refactored SQL generation for scalar functions.
 * #79: Added error builder to the project.
 * #81: Updated to the latest virtual-schema-common-java.
+* #82: Migrate common tools for implementing a new dialect from the `virtual-schemas` repository.
```
Comment (chiaradiamarcelo) [inline doc/changes/changes_9.0.0.md:12]:
```suggestion
* #82: Migrated common tools for implementing a new dialect from the `virtual-schemas` repository.
```

**Review by chiaradiamarcelo — APPROVED** [review]

**Review by chiaradiamarcelo — DISMISSED** [review]

File: doc/development/implementing_mandatory_sql_dialect_classes.md [hunk 2]
```diff
@@ -0,0 +1,418 @@
+# Implementing the SQL Dialect Adapter's Mandatory Classes 
+
+Each SQL Dialect adapter consists of two or more classes, depending on how standard-compliant the source behaves.
+The minimum implementation requires that you create the following mandatory classes:  
+  
+1. [`<Your_dialect_name>SqlDialect.java`](#creating-main-dialect-class)
+1. [`<Your_dialect_name>AdapterFactory.java`](#creating-adapter-factory)
+1. [`<Your_dialect_name>Adapter.java`](#creating-adapter)
```
Comment (chiaradiamarcelo) [inline doc/development/implementing_mandatory_sql_dialect_classes.md:8]:
```suggestion
1. [`<Your_dialect_name>JDBCAdapter.java`](#creating-adapter)
```

File: doc/development/implementing_mandatory_sql_dialect_classes.md [hunk 3]
```diff
@@ -0,0 +1,418 @@
+# Implementing the SQL Dialect Adapter's Mandatory Classes 
+
+Each SQL Dialect adapter consists of two or more classes, depending on how standard-compliant the source behaves.
+The minimum implementation requires that you create the following mandatory classes:  
+  
+1. [`<Your_dialect_name>SqlDialect.java`](#creating-main-dialect-class)
+1. [`<Your_dialect_name>AdapterFactory.java`](#creating-adapter-factory)
```
Comment (chiaradiamarcelo) [inline doc/development/implementing_mandatory_sql_dialect_classes.md:7]:
```suggestion
1. [`<Your_dialect_name>JDBCAdapterFactory.java`](#creating-adapter-factory)
```

File: doc/development/implementing_mandatory_sql_dialect_classes.md [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+1. [`<Your_dialect_name>Adapter.java`](#creating-adapter)
+
+[Athena Virtual Schema Repository](https://github.com/exasol/athena-virtual-schema)
```
Comment (chiaradiamarcelo) [inline doc/development/implementing_mandatory_sql_dialect_classes.md:10]: this line here seems a bit out of context. I'm not sure if something is missing or the line should be removed.

**Review by chiaradiamarcelo — APPROVED** [review]

File: doc/development/implementing_mandatory_sql_dialect_classes.md (hunk 2 above)
Comment (AnastasiiaSergienko) [reply doc/development/implementing_mandatory_sql_dialect_classes.md:8]: I like it more without JDBC

**Pull request merged**